addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/XxeScanRule.java

```diff
@@ -267,11 +270,58 @@ private void localFileReflectionAttack(HttpMessage msg) {
                             .setEvidence(matcher.group())
                             .setMessage(msg)
                             .raise();
+                    alertRaised = true;
                 }
                 if (isStop()) {
                     return;
                 }
             }
+
+            // Now if no issue is found yet, then we replace the values one at a time. Do this for a
+            // fixed
+            // number
+            // of Elements, depending on the strength at which the rule is used.
```

Comment:
spotlessApply is good at wrapping things but not un-wrapping them. Could you put 281-283 on a single line and redo the spotlessApply?